spec/helpers/models_helper.rb

```diff
@@ -0,0 +1,34 @@
+def create_and_activate_user(name)
+  user = User.create!(
+    name:, email: "#{name}@example.com", password: '123456', bio: "Bio of #{name}", photo: "#{name}.jpg"
+  )
+  user.confirm
+  user.save
+  user
+end
+
+def create_posts_for_user(user, count: 1)
+  posts = []
+  count.times do |i|
+    posts << Post.create!(
+      user:,
```

Comment:
- [ ] You will need a user to create the post.

Reply:
When we pass `user:user` rubocop asks us to omit the value